accounts-db/src/accounts_db.rs

```diff
@@ -7250,7 +7250,7 @@ impl AccountsDb {
                     // so, build a file name:
                     let hash = hasher.finish();
                     let file_name = format!(
-                        "{}.{}.{}.{}.{:016x}",
+                        "new.{}.{}.{}.{}.{:016x}",
```

Comment:
I'd prefer `.1` added to end. Make `.1` a version constant
`const VERSION: u64 = 1;`
and pass VERSION for new {} arg at end.
`new` ages itself ;-)

Reply:
yeah. done.